src/config/server.rs

```diff
@@ -162,7 +162,7 @@ impl Server {
         // the `script` in `public/github-redirect.html`
         let content_security_policy = format!(
             "default-src 'self'; \
-            connect-src 'self' *.ingest.sentry.io https://docs.rs https://play.rust-lang.org {cdn_domain}; \
+            connect-src 'self' https://crates.io https://staging.crates.io *.ingest.sentry.io https://docs.rs https://play.rust-lang.org {cdn_domain}; \
```

Comment:
Is this necessary?

Reply:
otherwise the CSP on https://crates.io would prevent the Swagger UI from sending HTTP requests to https://staging.crates.io if that server is selected.

if we don't want to allow that then the CSP change isn't necessary.

Reply:
this is obsolete now, see https://github.com/rust-lang/crates.io/pull/10186#issuecomment-2538377449